internal/operator-controller/rukpak/preflights/crdupgradesafety/checks.go

```diff
@@ -242,3 +242,13 @@ func Type(diff FieldDiff) (bool, error) {
 
 	return isHandled(diff, reset), err
 }
+
+// A change in a description is always considered safe and non-breaking.
```

Comment:
```suggestion
// Description differences should be allowed. Otherwise, we will block upgrades due to changes in the description, which will be considered as a bug from the end user's perspective. 
```

Reply:
@anik120 ^ I would suggest something
But we beed start the comment with Description
We are not using the lint check that catch it, but that is a better practice to doc funcs.

Reply:
That's a good point @camilamacedo86. I've changed the sentence around a little bit, PTAL, thank you!